README.rst

```diff
@@ -95,23 +95,39 @@ For detailed instructions on installing dependencies, see `Installing Dependenci
 Quick Cyclus Installation
 *************************
 
```

Comment:
```suggestion
The quickest way to install Cyclus and its dependencies relies on using the `conda-forge` channel and the `conda` package manager within the Anaconda python environment.  The following instructions guide you through that approach.
```

Reply:
Do we really want to advertise 'conda' as the quickest way to install cyclus deps ?!??

#swearing

Reply:
It is what it is... and it's probably true because conda let's you specify version numbers for each package.